src/test/resources/features/key_worker_details.feature

```diff
@@ -5,41 +5,12 @@ Feature: Key worker details
 
```

Comment:
I deleted the key_worker_available.feature file so it looks as tho the merge has created an empty file (tried to attach the comment to the availble.feature file but it wouldn't let me

Reply:
Ah yes i noticed that, result of a merge .. have just pushed update